### DISCUSSION TITLE
Make Sure to Double-Check the Network Before Deploying Contracts

### DISCUSSION THREAD

**Discussion by jalilbm** [Cyfrin/foundry-full-course-cu#1010] [General]
Hey everyone 👋,

I recently ran into an issue while following the course at lesson 6 and thought it would be helpful to share so others can avoid making the same mistake.

**The Issue:**
I was setting up my Alchemy account and noticed that I already had an app configured. Without paying much attention, I went ahead and used it. Turns out, the app was set to use the Ethereum Mainnet, not the SEPOLIA Testnet as in the tutorial.

**Why This is a Problem:**
If I had real ETH in that wallet (and hopefully I didn't), I would've spent it on deploying a test smart contract, which is definitely not what anyone wants.

**The Solution:**
Always double-check the network you're connected to, especially when you're just testing or following a tutorial. If the tutorial uses a testnet, make sure your setup also targets the same testnet.

Stay safe and happy coding! 🚀

**Comment by cromewar** [comment]
Thanks for this @jalilbm 
Also another tip is to always use a different wallet for testnet and never ever put real funds on it.
